bug-reproduction/Main.hs

```diff
@@ -61,8 +59,9 @@ main = do
         pure $ totals <> storyTotals
 
     totals <- foldM accumulateStory mempty tasks
-    liftIO $ putStrLn $ unlines
-      [ "Unknown: " <> show (length $ tUnknown totals)
-      , "Reproduced: " <> show (length $ tReproduced totals)
-      , "Not reproduced: " <> show (length $ tNotReproduced totals)
+    hPutBuilder stdout . getUtf8Builder $ foldMap
```

Comment:
How did you decide when it was OK to log vs having to maintain this plain-style output? And in the latter case, why not just `import Prelude.putStrLn` and keep the existing code?

Reply:
> How did you decide when it was OK to log vs having to maintain this plain-style output? 

In my eyes logs are for understanding the output of a program. They are contextual, but if you `/dev/null` them then you won't lose anything. This output is the result of the program and thus not for building context, but rather the things that the logs attempt to contextualize.

> why not just import Prelude.putStrLn and keep the existing code?

I figured I'd try to stay within the bounds of `rio` and not pull in `putStrLn`, since its absence is to discouraged its use.

Reply:
> In my eyes...

Hmm. I agree with the context/result distinction for sure. I think I'd just approach it more practically than principled-ly. In other words, you need to skip the logger for output that you need more control over (i.e. your *result*, which might be fed to something else if you were being a good unix filter). I usually write CLI tools to log to `stderr` and produce results to `stdout`; that's really the only option for a logging CLI tool meant to pipe anyway.

Personally, this tool is not going to see usage in a shell pipe-line, and we're not going so far as to split the handles (right?), so I would probably just go logging all the way and not be bothered by a little `[info] ` in front of my program results in exchange for the simplicity and reduction in decisions required. But I'm not suggesting or requesting that -- you do you.

> its absence is to discouraged its use

FWIW, I think you're stance is just in opposition to this idea generally and you should own it. In other words, if you're going to not use logging for this kind of output, then you've already departed with their philosophy and I would just pull in the simpler function -- rather than like *half*-following their nudge. Up to you though.